src/routes/+page.svelte

```diff
@@ -139,7 +139,12 @@
 										{formatDateAndTime(lastMessage.timestamp)}
 									</span>
 								</div>
-								<p class="message text-serif">
+								<p
+									class={`message text-serif ${
+										lastMessage.fromAddress === $profile.address ? 'my-message' : ''
+									}`}
+								>
+									{lastMessage.fromAddress === $profile.address ? 'You: ' : ''}
```

Comment:
We should extract this check into svelte constant `{@const myMessage=lastMessage.fromAddress === $profile.address}`